test/WufooTranslator_test.js

```diff
@@ -53,7 +51,8 @@ describe('WufooTranslator', function() {
             ID: 'Field1'
           }]
         })
-      }, function(err, sub) {
+      }, function (err, sub) {
+        should(err).not.exist
```

Comment:
Expected an assignment or function call and instead saw an expression.<br>Missing semicolon.